datasetinsights/io/downloader/base.py

```diff
@@ -0,0 +1,45 @@
+import re
+from abc import ABCMeta, abstractmethod
+
+
+class DownloaderRegistry(ABCMeta):
+    """ This class registers DatasetDownloader
```

Comment:
Please improve this docstring to make it clear for users what the `Registry` is used.